### PR TITLE
Updated some links in contracts.rst

### PR DESCRIPTION
**Pull request by sandakersmann**
Updated some links in contracts.rst

### CODE REVIEW

File: docs/contracts.rst [hunk 1]
```diff
@@ -982,8 +982,8 @@ contract, and a regular ``JUMP`` call will be used instead of a ``DELEGATECALL``
 .. index:: using for, set
 
 The following example illustrates how to use libraries (but
-be sure to check out :ref:`using for <using-for>` for a
-more advanced example to implement a set).
+be sure to check out `using-for <https://solidity.readthedocs.io/en/latest/contracts.html#using-for>`_
```
Comment (pirapira) [inline docs/contracts.rst:985]: I don't think `solidity.readthedocs.io` should appear in the documentation.  An older version of this manual should not link to the newest version like this.

Before this change, [the link is working already](https://solidity.readthedocs.io/en/latest/contracts.html#libraries).

File: docs/contracts.rst [hunk 2]
```diff
@@ -1114,9 +1114,9 @@ custom types without the overhead of external function calls:
 
 As the compiler cannot know where the library will be
 deployed at, these addresses have to be filled into the
-final bytecode by a linker
-(see :ref:`commandline-compiler` for how to use the
-commandline compiler for linking). If the addresses are not
+final bytecode by a linker (see 
+`using-the-compiler <https://solidity.readthedocs.io/en/latest/using-the-compiler.html#using-the-compiler>`_
```
Comment (pirapira) [inline docs/contracts.rst:1118]: The absolute URL should not appear.  Use the `:ref:` role.

File: docs/contracts.rst [hunk 3]
```diff
@@ -1160,8 +1160,9 @@ be lifted to the global scope later, so that by including
 a module, its data types including library functions are
 available without having to add further code.
 
-Let us rewrite the set example from the
-:ref:`libraries` in this way::
+Let us rewrite the set example from the 
+`libraries <https://solidity.readthedocs.io/en/latest/contracts.html#libraries>`_
```
Comment (pirapira) [inline docs/contracts.rst:1164]: URLs should not appear.  Use the `:ref:` role.

**Review by pirapira — CHANGES_REQUESTED** [review]

**Comment by chriseth** [comment]
Thanks for your contribution, but the links are working perfectly as they are.

**Pull request closed without merging**